### PR TITLE
update deps and generate clients for useroauthaccesstokens

### PR DESCRIPTION
**Pull request by stlaz**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -12,3 +12,5 @@ require (
 	k8s.io/code-generator v0.19.2
 	k8s.io/klog/v2 v2.3.0 // indirect
 )
+
+replace github.com/openshift/api => github.com/stlaz/api v0.0.0-20201112122008-c729dd5d5b82
```
Comment (soltysh) [inline go.mod:16]: This is wrong, was this already merged to openshift/api?

**Review by soltysh — CHANGES_REQUESTED** [review]
/hold
for wrong dependency fix

File: go.mod (hunk 1 above)
Comment (stlaz) [reply go.mod:16]: it hasn't, I completely forgot about this, had it locally for some time. Glad you caught it.

**Comment by stlaz** [comment]
depends on https://github.com/openshift/api/pull/788

**Comment by stlaz** [comment]
@soltysh please see if the PR is to your liking now :)

**Review by soltysh — APPROVED** [review]
/lgtm
/approve
/hold cancel

**Pull request merged**